src/main/java/bio/terra/service/dataset/flight/create/CreateDatasetAssetStep.java

```diff
@@ -92,24 +90,35 @@ public StepResult doStep(FlightContext context) {
         } catch (Exception e) {
```

Comment:
Not your code, but I'm curious why the fault is surrounded by a try catch. It catches its own exception and then retrhows it. Seems odd.